Makefile

```diff
@@ -50,16 +50,20 @@ lint-js:
 	@yarn eslint --ext js --report-unused-disable-directives "**/*.js"
 	@yarn prettier --check "**/*.{js,ts,md}"
 lint-python:
-	@python3 -m flake8 ./
+	@python3 -m flake8p ./
 lint-html:
 	@yarn htmlhint "testCourse/**/question.html" "exampleCourse/**/question.html"
 lint-links:
 	@node tools/validate-links.mjs
 
-format: format-js
+format: format-js format-python
 format-js:
 	@yarn eslint --ext js --fix "**/*.js"
 	@yarn prettier --write "**/*.{js,ts,md}"
+format-python:
+# Very annoying that isort doesn't allow this to be set in the config
+	@python3 -m isort ./elements/**/*.py
```

Comment:
Rather than doing a positive match, could we instead run `isort ./` and set `skip_glob` in the config file? I'm pretty sure that this is the semantics we want anyways; IMO, linting + formatting should be opt-out, not opt-in. Ditto for the other tools, e.g. Black, though I think we can leave Pyright as opt-in for now.

Reply:
That makes sense I think. The linting might need to be opt-in until more files can be made compliant, but we can just always run isort and black over everything

Reply:
@nwalters512 set up negative matching for autoformatting in latest push! I think we can change the linting to use this once a few more files are brought into compliance. For now, the autoformatter is run over the `elements`, `python`, and `graders` directories, so the later PR to turn on linting for those should be shorter (since it won't include running the autoformatting itself).